pal/system-info/native/system-info.ts

```diff
@@ -76,12 +76,17 @@ class SystemInfo extends EventTarget {
 
         this.pixelRatio = jsb.device.getDevicePixelRatio() || 1;
 
-        // init capability
-        this.supportCapability = {
-            webp: true,
-            gl: true,
-            canvas: true,
-            imageBitmap: false,
+        this._featureMap = {
+            [Feature.CANVAS]: true,
+            [Feature.GL]: true,
+            [Feature.WEBGL2]: false,
+            [Feature.WEBP]: true,
+            [Feature.IMAGE_BIT_MAP]: false,
+
+            [Feature.INPUT_KEYBOARD]: !this.isMobile,
+            [Feature.INPUT_MOUSE]: !this.isMobile,
+            [Feature.INPUT_TOUCH]: this.isMobile,
```

Comment:
We always support touch input on desktop devices (using mouse event to simulate), it's very convenient for user to design mouse and touch compatible event handlers.

Besides, there are many desktop devices have touch screen nowadays, like Microsoft Surface and Huawei notebooks.

Simply set true here is backward compatible